### PR TITLE
Windows CI in GitHub Actions

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -99,3 +99,6 @@ tags
 build/
 install/
 log/
+
+### Visual Studio ###
+.vs
```
Comment (jparisu) [inline .gitignore:104]: EOF line break

File: CMakeLists.txt [hunk 2]
```diff
@@ -331,21 +331,23 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE
...
 # Test
 ###############################################################################
-option(BUILD_TESTS "Build Fast DDS Monitor library and documentation tests" OFF)
+option(BUILD_TESTS "Build Fast DDS Monitor appliction and documentation tests" OFF)
+option(BUILD_APP_TESTS "Build Fast DDS Monitor application tests" OFF)
 option(BUILD_DOCUMENTATION_TESTS "Build Fast DDS Monitor documentation tests" OFF)
 
 if (BUILD_TESTS)
+    set(BUILD_APP_TESTS ON)
     set(BUILD_DOCUMENTATION_TESTS ON)
```
Comment (jparisu) [inline CMakeLists.txt:340]: NIT: This statement is repeated in `docs/CMakeLists.txt` in line 52.

**Review by jparisu — CHANGES_REQUESTED** [review]
Minor changes

File: CMakeLists.txt (hunk 2 above)
Comment (rsanchez15) [reply CMakeLists.txt:340]: Yes, that's because you can compile the documentation without compiling the monitor.

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**